### PR TITLE
feat: cloudy improvements [part 1]

### PR DESCRIPTION
**Pull request by IrakliJani**
closes: https://github.com/lightdash/lightdash-internal-work/issues/1573
closes: https://github.com/lightdash/lightdash-internal-work/issues/1401

copy:

CleanShot 2024-03-13 at 02 27 54@2x
CleanShot 2024-03-13 at 02 45 12@2x

link bug:

CleanShot 2024-03-13 at 02 50 13@2x

CleanShot 2024-03-13 at 02 50 19@2x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,23 +13,23 @@ You can develop Cloudy locally by following these steps:
...
 
 SLACK_STATE_SECRET=this can be anything
 
-SLACK_AUTHORIzED_TEAMS=T0163M87MB9,T03942PB7E3
+SLACK_AUTHORIZED_TEAMS=T0163M87MB9,T03942PB7E3
 
 PG_CONNECTION_STRING=postgres://user:pass@host:port/database
 
-SITE_ENV=https://domain.where.app.hosted.com
+SITE_ENV=https://cloudy-dev-crazy-cayote.loca.lt
```
Comment (IrakliJani) [inline README.md:32]: Removed `ngrok` and added `localtunnel` because, with the latter, you can keep using the same subdomain.

File: index.js [hunk 2]
```diff
@@ -40,26 +40,28 @@ const expressReceiver = new ExpressReceiver({
...
     storeInstallation: createInstallation,
     fetchInstallation: getInstallation,
     deleteInstallation,
   },
-})
+});
+
 const app = new App({
   receiver: expressReceiver,
+  logLevel: process.env.ENV === 'development' ? 'DEBUG' : 'INFO',
```
Comment (IrakliJani) [inline index.js:64]: this is quite useful when debugging slack api calls and responses

File: README.md [hunk 3]
```diff
@@ -52,28 +52,29 @@ yarn start
 ```
 
 ### 2. Run ngrok and update URLs
```
Comment (rephus) [inline README.md:54]:
```suggestion
### 2. Run localtunnel and update URLs
```

File: README.md [hunk 4]
```diff
@@ -52,28 +52,29 @@ yarn start
...
+
 ```shell
 yarn expose
 ```
 
 Now you need to update the following URLs with the last `Forwarding` url that ngrok returns:
```
Comment (rephus) [inline README.md:60]:
```suggestion
Now you need to update the following URLs with the last `Forwarding` url that localtunnel returns:
```

File: README.md [hunk 5]
```diff
@@ -13,23 +13,23 @@ You can develop Cloudy locally by following these steps:
 git clone https://github.com/lightdash/lightdash-cloud-announcer.git
 ```
 
-
 ### Make sure you create and set an `.env` file correctly.
 
+ℹ️ app id for "dev Cloudy" is `A03NM81NBFD`
 
-SLACK_SIGNING_SECRET=[Get variable from slack app settings](https://api.slack.com/apps/A033CLM638C/general)
+SLACK_SIGNING_SECRET=[Get variable from slack app settings](https://api.slack.com/apps/[APP_ID]/general)
```
Comment (rephus) [inline README.md:20]: will these URL work with [APP_ID] ?

**Review by rephus — APPROVED** [review]
code look good to me, I was not able to test this locally , also I don't have access to cloudy app https://api.slack.com/apps/A03NM81NBFD/general

**Pull request merged**